Tests/MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -204,7 +181,7 @@ class CarPlayManagerTests: XCTestCase {
             CLLocationCoordinate2D(latitude: 34.054081, longitude: -118.243412),
         ])
         choice.userInfo = (Fixture.route(from: "route-with-banner-instructions", options: options), 0, options)
-
+        CarPlayMapViewController.swizzleMethods()
```

Comment:
I wonder if we should un-swizzle at the end of execution to prevent any other side-effects (for `UNUserNotificationCenter` as well)?

Reply:
Good idea, thank you!